src/ui/Forms/Translate/AutoTranslate.cs

```diff
@@ -1393,7 +1393,7 @@ private void buttonOK_Click(object sender, EventArgs e)
 
         private IAutoTranslator GetCurrentEngine()
         {
-            return _autoTranslatorEngines.First(p => p.Name == nikseComboBoxEngine.Text);
+            return (IAutoTranslator)nikseComboBoxEngine.SelectedItem;
```

Comment:
Faster current engine selection, since we already have the instance in `nikseComboBoxEngine.SelectedItem`